index.html

```diff
@@ -47,6 +50,7 @@ <h1 class="logo"><em>HotMaps</em></h1>
       </div>
     </li>
   </ul>
+  </div>
   <a href="#" data-activates="slide-out" class="button-collapse"><i class="material-icons">menu</i></a>
```

Comment:
See latest in develop branch. Matt had to change some stuff, so there is no longer a side panel but rather a grid system that acts like a fixed side panel